modules/apps/headless/headless-builder/headless-builder-impl/src/main/java/com/liferay/headless/builder/internal/application/endpoint/EndpointMatcher.java

```diff
@@ -74,12 +75,12 @@ private String _getSingleElementRegex(APIApplication.Endpoint endpoint) {
 
 		if (Objects.equals(
 				endpoint.getPathParameter(),
-				APIApplication.Endpoint.PathParameter.ID)) {
+				HeadlessBuilderConstants.PATH_PARAMETER_ID)) {
 
 			return endpointPath + "\\d+";
 		}
 
-		return endpointPath + "\\D+";
+		return endpointPath + ".+";
```

Comment:
I think we talked about including accepting the everything except a slash / , am I wrong?

Reply:
@ccorreagg  Correct me if I'm wrong, I was making some test cases in which I insert a path parameter value including a `/`. The problem comes from that this is not going to been validated in the EndpointMatcher, this is because when I try to insert a value like: `/students/{strudentName}` and I pass this ->  "example/example", JAX-RS thinks that there is a new path called `/students/example/example` and it will fail before matching the EndpointMatcher implementation. So in that case, should we maintain this validation?

Reply:
I you want we can see that next week